test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -1574,20 +1916,21 @@ def tearDown(self):
         conf = {'enabled': 'False', 'freq': 30, 'event': events}
         self.server.manager(MGR_CMD_SET, HOOK, conf, self.hook_name)
         # Cleanup any temp file created
-        self.logger.info("Deleting temporary files %s" % self.tempfile)
-        self.du.rm(hostname=self.serverA, path=self.tempfile,
-                   force=True)
+        self.logger.info('Deleting temporary files %s' % self.tempfile)
+        self.du.rm(hostname=self.serverA, path=self.tempfile, force=True)
         # Cleanup frozen jobs
         if 'freezer' in self.paths:
-            self.logger.info("Cleaning up frozen jobs ****")
+            self.logger.info('Cleaning up frozen jobs ****')
             fdir = self.paths['freezer']
             if os.path.isdir(fdir):
+                self.logger.info('freezer directory present')
                 fpath = os.path.join(fdir, 'PtlPbs')
                 if os.path.isdir(fpath):
-                    jid = glob(os.path.join(fpath, "*", ""))
+                    jid = glob.glob(os.path.join(fpath, '*', ''))
+                    self.logger.info('found jobs %s' % jid)
                     if jid:
                         for files in jid:
-                            self.logger.info("*** found jobdir %s" % files)
+                            self.logger.info('*** found jobdir %s' % files)
```

Comment:
Same here.

Reply:
Same here, I'd prefer to leave it. These aren't new messages.